### ISSUE TITLE
only works in inbox?

### ISSUE THREAD

**Issue by watata84** [teresa-ou/inboxy#127]
not work in other folders right ?

**Comment by vtexperts** [comment]
Correct, it works in INBOX and each category tab (Primary, Promotions, Social, Updates, Forums) within INBOX.

If you are "in the folder" e.g label:folderABC, the emails will not be grouped, because you have the folder open.

**Comment by benoror** [comment]
Hi! Correct for the most part: bundling targets **inbox-style message lists** (including non-default inbox types in Inbundly v3.2.0). Plain label folders and search results are not bundled the same way.

**View all** from a bundle jumps to the label search. If you need bundling in more views, please open a feature request with which left-nav entries matter.

Chrome: https://chromewebstore.google.com/detail/inbundly/pbfjicjmcpogjlbpljebhhgkgfhbdcga
Site: https://inbundly.com/
Issues: https://github.com/benoror/inbundly/issues/new
